src/libs/actions/IOU.js

```diff
@@ -116,7 +116,7 @@ function buildOnyxDataForMoneyRequest(
             key: `${ONYXKEYS.COLLECTION.REPORT}${iouReport.reportID}`,
             value: {
                 ...iouReport,
-                lastMessageText: iouAction.message[0].text,
+                lastMessageText: ReportUtils.formatReportLastMessageText(iouAction.message[0].text),
```

Comment:
>  Can't we just prevent wrap here?

Are you mean remove new lines "\n" from message text?
If you mean that the lastMessageText will not be inconsistent with backend, because backend return only first line of message text.

Reply:
the wrap happened here because `numberOfLines={1}` and `whiteSpace: 'pre'` does not work as expected when <Text/> contains `\n`

For that, we can fix it by replace `styles.pre` with `styles.noWrap` here
https://github.com/Expensify/App/blob/4362fe76cee9450becc3f9f6de520955c94f8a71/src/components/LHNOptionsList/OptionRowLHN.js#L87-L92

This will remove the new lines "\n" from message text and fix the wrap issue, but inconsistency with backend will still issue.

Reply:
I think if we fix the inconsistency bug, the wrap bug will disappear so no need for this change.
What do you think @aimane-chnaif ?

Reply:
Ok, are you agree to using styles.noWrap?

Reply:
Inconsistent behavior for `styles.noWrap` in android and ios native.
Example: `line1\nline2\nline3\nline4`

![Screenshot 2023-08-25 at 6 26 57 PM](https://github.com/Expensify/App/assets/41129870/0dd1acd9-6730-4a10-8fad-080785ecde53)

Reply:
All platforms work fine similar android screenshot, only native iOS displays the first line

Reply:
can we ignore native ios?

Reply:
This is the quick fix for me now `.replace(/(\r\n|\n|\r) */gm, " ")` it seems a workaround, but it works fine.
Do you agree?, or we will keep wrap bug in separate issue?

Reply:
I can't find a quick fix right now. What is the next step now?

- use `styles.noWrap` and fix wrap issue partially with inconsistent result on iOS.
- do nothing and keep wrap issue separately?

Reply:
Yes, I update the code to use styles.noWrap

Reply:
`styles.noWrap` caused regression - https://github.com/Expensify/App/issues/27586